### PR TITLE
Friendly urls

### PR DESCRIPTION
**Pull request by brayhoward**
implemented friendly urls for users and blog post, via friendly_id gem.

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -7,6 +7,11 @@ def new
     @user = User.new
   end
 
+  def show
+    @username = User.find(params[:id])
```
Comment (elizamarcum) [inline app/controllers/users_controller.rb]: I don't believe this variable is used anywhere.

**Comment by elizabrock** [comment]
I think

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (erinpagemd) [inline app/controllers/users_controller.rb]: It has been removed.

**Comment by elizamarcum** [comment]
THIS IS HAPPENING!!!

**Comment by brayhoward** [comment]
### My feels!!  :+1:

**Pull request merged**